metadata_lookup.ipynb

```diff
@@ -0,0 +1,995 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    # Note: but it need to read the csv files again like when initializing metadata -&gt; cost run time. Looking for suggestion to improve the performance
</pre><p>You can store the DataFrame as a class attribute and only read it once on lazily on demand.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/pertpy-tutorials/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>